migrations/seeders/registration.py

```diff
@@ -0,0 +1,24 @@
+from faker import Faker
+from api.models import Registration as RegistrationModel
+from api import db
+
+
+class Registration:
+    faker = Faker()
+
+    def create_registration(
+        self, user_phone_number, system_phone_number, partner_id, user_id
+    ):
+        registration = RegistrationModel()
+        registration.user_id = user_id
+        registration.user_phone = user_phone_number
+        registration.system_phone = system_phone_number
+        registration.partner_id = partner_id
+
+        db.session.add(registration)
+        db.session.commit()
+
+
+if __name__ == "__main__":
+    seeder = Registration()
+    seeder.create_registration("+911234567890", "9876543210", 1, 1)
```

Comment:
I think this system phone number 9876543210 is not added in the system_phone seeder, because there it is getting generated randomly.

Reply:
It may cause issues while adding test case for registration, if there will be any dependency on the state number table